### ISSUE TITLE
Standardize DeFi content and categories across /defi and /dapps

### ISSUE THREAD

**Issue by 0xKai27** [ethereum/ethereum-org-website#9129]
## Description

The Defi [page](https://ethereum.org/en/defi/) contains multiple links to sample dapps which requires the user to be redirected to the dapps [page](https://ethereum.org/en/dapps/?category=finance). As part of this user journey, the flow between the pages could be optimised whereby:

- [Advanced trading](https://ethereum.org/en/defi/#trading) does not have any relevant category nor dapps that are being shown. The "Advanced trading" description also includes "limit orders" which is more suited to be under the exchanging of tokens. Propose to change "Advanced trading" to "Derivatives" for better framing with regards to TradFi and risk management strategies.
- [Exchange tokens](https://ethereum.org/en/defi/#swaps) has limited info around the different implementations (AMM vs order-book). Propose to add more content around the different types of DEXes and to link to relevant external links. Additionally, to add orderbook examples to the dapps page.
- `Trading and prediction markets` dapp category does not have a relevant description in the DeFi page. Propose to change naming to "Prediction markets" to avoid categorical confusion with "Token swaps". Consequently, to move `Loopring` under the "Token swaps" category instead.
- [Borrowing](https://ethereum.org/en/defi/#lending) and [Lending](https://ethereum.org/en/defi/#lending) on the DeFi page both have the same anchor tags (`/defi/#lending`) but there is a horizontal rule dividing the two. For consistency with the "Lending and borrowing" category under the dapps page, propose to remove horizontal rule and rename section as "Lending and borrowing"
- [Aggregators and portfolio managers](https://ethereum.org/en/defi/#aggregators) leads users to a "See portfolio dapps" button which is showing portfolio dapps with native aggregators. This might result in the blurring between 2 distinct concepts as aggregators refers more to dapps which build on top of DeFi primitives in order to maximise returns through implementing specific strategies (as opposed to just UI interfaces for portfolio management). Propose to create a new category for aggregators which will include demand-side aggregators (1inch, matcha), supply-side aggregators (yearn, beefy), as well as governance aggregators (convex). Portfolio management dapps can have their own native aggregator or build on top of a separate third-party aggregator.

## Other considerations

- To decide desirability of adding a section on "Bridges", be it between different Layer 1s or between Layer 1 & 2

**Comment by corwintines** [comment]
@0xKai27 what category do you see this under?

Also, would you like to lead adding Yearn?

I think this project is fine to add.

**Comment by corwintines** [comment]
Hey @0xKai27.

I agree that we should align these pages a bit more. What I think would be best, is first getting these new categories implemented from your suggestions in https://github.com/ethereum/ethereum-org-website/issues/9146, and then updating this content around these new categories. What do you think?

**Comment by 0xKai27** [comment]
Yes, we should definitely land out #9146 first then it'll make the distinctions much clearer

**Comment by corwintines** [comment]
Great, I responded in #9146, happy to let you start working on that and then we can come back to this

**Comment by corwintines** [comment]
Ive created a pass at the defi page as part of our content audit
